pkg/controller/controller_utils.go

```diff
@@ -83,6 +83,9 @@ const (
 	// The number of batches is given by:
 	//      1+floor(log_2(ceil(N/SlowStartInitialBatchSize)))
 	SlowStartInitialBatchSize = 1
+
+	// NodeNameKeyIndex is the name of the index used by PodInformer to index pods by their node name.
+	NodeNameKeyIndex = "spec.nodeName"
```

Comment:
nit: maybe let's call it "PodNodeNameKeyIndex" ?

Reply:
Done.